### PR TITLE
Netconfig fix

### PR DESCRIPTION
**Pull request by alpinesnow**
changed config-tutorial.json to tutorial-netconfig.json

### CODE REVIEW

File: Exercise4/README.md [hunk 1]
```diff
@@ -90,22 +90,30 @@ Server$ docker kill onos
...
 
 1. Download the files for Exercise 4 or clone the repository, as described in
    [Software Used in Tutorial](../README.md#software-used-in-tutorial).
-2. Use the following command to pull and start ONOS with the PINS driver and SAI
+2. Go to the directory with the configuration files for Exercise4. Ensure that
+   the shell script is executable.
+
+    ```
+    Server$ cd $TUTORIALS_PATH/Exercise4
+    Server$ chmod 0755 flow-objectives.sh
```
Comment (bocon13) [inline Exercise4/README.md:106]: If they clone the repo, it should be done for them right??

File: Exercise3/README.md [hunk 2]
```diff
@@ -78,35 +78,38 @@ If you reload the configuration on your switches or reboot them, review Exercise
...
+    ```
+    Server$ wget https://github.com/pins/p4rt-client/releases/download/202112/p4rt-client
+    Server$ wget https://github.com/pins/p4rt-client/releases/download/202112/macToIpV6
+    ```
+
+4. Make sure the script and programs are executable.
+
+    ```
+    Server$ cd $TUTORIALS_PATH/Exercise3
+    Server$ chmod 0755 p4rt-client-setup.sh p4rt-client macToIpV6
```
Comment (bocon13) [inline Exercise3/README.md:111]: If they clone the repo, it should be done for them, right?

File: Exercise4/README.md (hunk 1 above)
Comment (alpinesnow) [reply Exercise4/README.md:106]: yes, I'll take it out. If you do anything other than clone, you should know how to run a shell script.

File: Exercise3/README.md (hunk 2 above)
Comment (alpinesnow) [reply Exercise3/README.md:111]: same comment. I'll change it.

**Pull request merged**